### DISCUSSION TITLE
Feature Request: Neon DB Integration

### DISCUSSION THREAD

**Discussion by melnikovic** [coollabsio/coolify#4541] [Feature Requests]
Hello Coolify team,

First of all, thank you for creating such an amazing platform! I would love to see Neon DB added as a feature or service in Coolify.

Neon DB is a modern cloud-native Postgres solution that offers features like serverless scaling and branching. It would be a great addition for developers who are looking for a fast and scalable database option. More info here: [Neon DB](https://neon.tech/home).

Additionally, Neon provides official Docker images that could be integrated:

[Neon Database Docker Images](https://registry.hub.docker.com/r/neondatabase/neon/tags)
[Neon Compute Node Docker Images](https://registry.hub.docker.com/r/neondatabase/compute-node/tags)
I believe adding Neon DB would be a great enhancement to Coolify and could attract more users who want to leverage its modern database capabilities.

Let me know if there’s anything I can do to help with this!

Thank you for considering this request.

**Comment by wottpal** [comment]
Would be so dope!

There is this https://coolify.io/docs/services/neon-ws-proxy template, but I guess that's something else and not full Neon right? Even though the docs kinda suggest it's neon: https://coolify.io/docs/services/neon-ws-proxy cc @peaklabs-dev

**Comment by D3m0n92** [comment]
Any news on this?

**Comment by TheDanniCraft** [comment]
+1 Would love to see this. As far as I know there isn't a way to have a separate database (branch) for each preview build

**Comment by gltched-usr** [comment]
+1 +1 +1 +1 +1 +1
This would be so usefull for preview deplyoments
